pkg/release/condition_set.go

```diff
@@ -61,3 +74,25 @@ func (c *conditionSet) CRD(ctx context.Context, crd *apiextensionsv1beta1.Custom
 		return nil
 	}
 }
+
+func (c *conditionSet) Secret(ctx context.Context, namespace, name string) ConditionFn {
```

Comment:
The implication in naming so far was that this always means `secret exists` or `CRD exists` - I think this might be a little confusing for someone looking at the code currently. They will just see condition `secret`. Would it maybe make sense to rename functions to `SecretExists`? That way they would state a condition in the name.

Reply:
I just based that on current CRD name.